modules/olmv1-about-target-versions.adoc

```diff
@@ -29,6 +29,8 @@ metadata:
 spec:
   packageName: openshift-pipelines-operator-rh
   installNamespace: <namespace_name>
+  serviceAccount:
+    name: <service_account>
```

Comment:
Since this var is being added now, does it need a call out and description?

Reply:
I am going to add an xref to the supporting documentation on service accounts once that content merges. I debated adding a call out here for the sake of completeness. When I tried it, it actually distracted from the point of the module, which is focused on upgrade edges.

Reply:
Anyhow, that is just to say that I think your suggestion is a good idea, but it didn't work for this content in the end. Thanks, @xenolinux!